tests/e2e/local/minikube/install_prereqs.sh

```diff
@@ -4,10 +4,16 @@ case "${OSTYPE}" in
   darwin*) sh install_prereqs_macos.sh;;
   linux*)
     DISTRO="$(lsb_release -i -s)"
+    # If lsb_release is not installed on CentOS, DISTRO will be empty.
+    if [ -z "$DISTRO" && -f /etc/centos-release ]; then
```

Comment:
in the future or as a followup PR, a more generic redhat-release may make more sense (if that file is still present in centos).

Reply:
Yes, I will fix the redhat support in follow-up.